### ISSUE TITLE
DOSBOX-X 20230501 Windows installers

### ISSUE THREAD

**Issue by phrossignol** [joncampbell123/dosbox-x#4257]
### Question

Hello,
I get an error 404 when trying to download Windows 32 and 64-bits installers.
Thanks for some help ...

### Have you checked that no similar question(s) exist?

- [X] I have searched and didn't find any similar question.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by Torinde** [comment]
I think they weren't created this time.

The WinXP installer works on subsequent Windows versions as well.

As for the Vista+ installers - I wonder if those can be made automatically buildable on every commit/PR - to get intermediate artifacts between the official releases... e.g. whatever extra actions are needed on release time to be automated?

**Comment by maron2000** [comment]
@Dobby233Liu Maybe temporarily amend the link to the portable version as we did before?

**Comment by Dobby233Liu** [comment]
> @Dobby233Liu Maybe temporarily amend the link to the portable version as we did before?

Sorry for missing this. I think we should probably not pretend that the portable packages are the installers. Maybe we should add extra links to the portable packages on the homepage

**Comment by maron2000** [comment]
How about simply mention that it's "Currently not available"?

**Comment by jharris1993** [comment]
Is there either a way to get this, an older version, or a ETA?

Thanks!

**Comment by grstatdoc-gr** [comment]
Okay, thanks. (I'll admit I have a hard time seeing if a question I wanted to ask was already asked before).

**Comment by maron2000** [comment]
@grstatdoc-gr  Yeah, my intention is NOT to blame you to open duplicate questions, but redirect you to an appropriate ticket so that we don't have to discuss same subject all over this place.

**Comment by jharris1993** [comment]
> Is there either a way to get this, an older version, or a ETA?
> 
> Thanks!

I was able to, (eventually), find the appropriate version after about 45 minutes of careful research. (It's at the ***bottom*** of the "Releases" page.)

It would be helpful if this were made more obvious.

**Comment by Dobby233Liu** [comment]
> How about simply mention that it's "Currently not available"?

What about this?

![image](https://github.com/joncampbell123/dosbox-x/assets/28653235/caf46c78-e5ff-43a2-a48b-6e246b65340a)

**Comment by maron2000** [comment]
Your suggestion will sure be fine.

**Comment by AnessZurba** [comment]
Hi, I'm trying to understand what is the conclusion here

Is the XP installer the recommended way to install Dosbox-x on Windows today? Or is it just a "workaround" until the missing installers are added?

**Comment by maron2000** [comment]
Unfortunately, the Windows installer is not available for the latest release.
You can try the portable builds or the nightly builds instead. 

The XP installer is available whether or not normal installer is available. 
You can also try it but it isn't the official recommended build for Vista+.

**Comment by AnessZurba** [comment]
Anything I can do to fix this? Do we know what the problem is?
Alternatively, how can I manually create the installers for myself?

**Comment by maron2000** [comment]
It is hopefully fixed by #4869.

**Comment by AnessZurba** [comment]
Thanks!
